test/test_evaluation/test_abstract_evaluator.py

```diff
@@ -294,11 +298,14 @@ def test_add_additional_components(self):
 
         context = BackendContext(
             temporary_directory=os.path.join(self.working_directory, 'tmp'),
+            output_directory=os.path.join(self.working_directory, 'tmp_output'),
```

Comment:
Do we need to pass in the output directory? Auto-sklearn doesn't use this concept any more.

Reply:
It's unfortunately a required argument, AutoPytorch doesn't use it either so it'll be fixed with the new backend of automl_common

Reply:
Funnily enough, it's typed as `Optional[str]` but doesn't provide a default parameter of `None` s it's required. It's updated in the new backend so I will leave it for now.